metrics-operator/main.go

```diff
@@ -177,17 +171,6 @@ func main() {
 	}
 }
 
-func serveMetrics() {
-	log.Printf("serving metrics at localhost:2222/metrics")
```

Comment:
Is the metrics URL location being logged somewhere? I think it's generally helpful to do so.

Reply:
it was, but only for the prometheus endpoint - i have added the path to the custom metrics api endpoint to the logs as well now